src/io/mod.rs

```diff
@@ -20,6 +20,7 @@ use crate::status::StatusBackend;
 pub mod filesystem;
 pub mod format_cache;
 pub mod itarbundle;
+pub mod cached_itarbundle;
 pub mod local_cache;
```

Comment:
Can we get rid of the `itarbundle` and `local_cache` modules after this change? If the worry is API breakage, I think I'd rather break things than keep the legacy code hanging around and confusing people — this is the 0.1.x version series, after all. (Although it might be fair to bump the version to 0.2.0 to indicate that there has been a significant API change.)

Reply:
Yes, that was my original goal. I hope that not many people are relying on such internals.